### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tangwenqiang**
add format guideline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # sharing
 sharing from my friends
+
+## 格式规范
+* 尽量用markdown来写
+* 每行文字不要太长，方便讨论 (add comments)
+* markdown素材文件尽量压缩不要太大，放在resources目录中，文件名称要有意义
```
Comment (tangwenqiang) [inline README.md:7]: 请新建一个resources目录

Comment (tangwenqiang) [reply README.md:7]: okay

Comment (tangwenqiang) [reply README.md:7]: > 请新建一个resources目录

okay, will create it later

**Review by tangwenqiang — COMMENTED** [review]
LGTM

**Comment by csuzhouzc** [comment]
agree

**Review by csuzhouzc — APPROVED** [review]
agree

**Pull request closed without merging**